boards/STM32F103ZET6-F103ZE-Board.md

```diff
@@ -308,40 +309,45 @@ title: "STM32F103ZET6 - F103ZE board"
 
 ## TFT
 
-| Pin   | Connected to |
-| ----- | ------------ |
-|       | NRST         |
-|       | Ground plane |
-|       | PD9          |
-|       | PD10         |
-|       | PE15         |
-|       | PD8          |
-|       | PE13         |
-|       | PE14         |
-|       | PE11         |
-|       | PE12         |
-|       | PE9          |
-|       | PE10         |
-|       | PE7          |
-|       | PE8          |
-|       | PD0          |
-|       | PD1          |
-|       | PD14         |
-|       | PD15         |
-|       | PD5          |
-|       | PD4          |
-|       | PG12         |
-|       | PG0          |
-|       | PB2          |
-|       | PB1          |
-|       | PF8          |
-|       | PF9          |
-|       | N.C.         |
-|       | N.C.         |
-|       | VBAT         |
-|       | Ground plane |
-|       | +3.3V rail   |
-|       | Ground plane |
+| Pin   | TFT  | Connected to | Function       |
```

Comment:
Thanks for adding the names of the pins on the TFT module. I am still in doubt whether these should be included here since they're not part of the board itself. The pins might have different names on other TFT modules. And this information is applicable to multiple boards. To be honest, the best place for this information might be a separate page for the display module.

Reply:
Most of the pins will probably have names that are similar on different display boards, but there will be differences. As you can see from the photo of the two boards I used for this testing, they don't even have the same number of pins (32 - 34). A separate page about displays would be useful, but not very easy to write: how much to say, how deep to dive into the matter. I would not want to write that on my own, but would be glad to participate....

Reply:
I can see why you don't want to write such page. I don't want to do it either. You're right that you can go very deep into the matter. Let's keep the scope of STM32-base focused on STM32 dev boards only.

I am still against having the TFT module-specific pin names in this table. On the other hand, this information is not covered by the lcdwiki website. Is having a mapping table a good alternative? It may also include the mapping for other TFT modules.

Reply:
Linking to lcdwiki is no problem, by the way. I think it is a very useful resource.

Reply:
Have removed the column for names on the TFT-side of the TFT connector. As you have already discovered I think we should try to make a "Connect a display page". I will have a go at it when (if?) we ever get THIS done.

Reply:
I've still got some thinking to do on this one, but for this PR it is fine right now.